.env.dev.tmpl

```diff
@@ -5,6 +5,5 @@ AUDIUS_URL=https://audius.co
 USER_NODE=https://usermetadata.audius.co
 LEGACY_USER_NODE=https://creatornode.audius.co
 IDENTITY_SERVICE=https://identityservice.staging.audius.co
-SEGMENT_IOS_WRITE_KEY=abc
-SEGMENT_ANDROID_WRITE_KEY=abc
+AMPLITUDE_WRITE_KEY=abc
```

Comment:
@jowlee I ran into an annoying crash issue because `.stage.env` was missing the `SEGMENT_IOS_WRITE_KEY` which I was gonna put a PR up to add to the template, but since you're replacing it here can you add `AMPLITUDE_WRITE_KEY` to the `.stage.env.tmpl` (if this has the same problem?)

Reply:
Maybe prod too, as long as it doesn't interfere with how those envs are set for actual deployments